R/model-validation.R

```diff
@@ -75,6 +75,9 @@ enw_score_nowcast <- function(nowcast, latest_obs, log = FALSE,
     scoringutils::check_forecasts(long_nowcast)
   }
 
+  long_nowcast[, prediction := as.numeric(prediction)]
```

Comment:
does this problem also not potentially crop up in the line above this for `check_forecasts()`?

Reply:
seems like advance this cast up, just in case?

Reply:
it doesn't as the problem was in an internal `scoringutils` function called for interval scoring. Your right and it may as well be moved up.